data_science/archive_exploration/02 - exploring trees.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Trees\n",
+    "Archives are stored as trees - a node may represent a _piece_ (eg an individual bit of the archive subject's correspondence), an _item_ (typical leaf node in the tree, usually a box containing multiple pieces), a _section_, _series_, _subsection_, _subsubseries_, _subsubsubseries_ etc etc etc (describing the informational hierarchy though which context can be attributed to items and the collection as a whole can be understood, until, at the top of the tree, we reach the _collection-level_ record. The CALM data contains many collection level records, some of which are also contextually linked. All of these nodes in the trees are stored as individual rows in the CALM `.json`, and its position in the tree is described by the `AltRefNo` field. Let's open some up and have a look at their structure."
```

Comment:
“its position” is a little ambiguous – is this the position of a node? There’s not a singular subject in the first half of the sentence; perhaps:

> Every node in the tree is stored as a single row in the CALM JSON export, and its position in the tree is described by its `AltRefNo` field.